cmd/onewire-list/main.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Onewire-list lists all onewire buses and devices. It furthermore treats
```

Comment:
It's conflating two separate tasks: one to list all the devices detected on onewire bus, the other to query a ds18b20. I think I'd prefer two single-purpose focused executables instead.

Reply:
Okay. So I removed the device specific part from this.